src/Room.cpp

```diff
@@ -83,3 +95,17 @@ sf::Texture Room::getTexture(){
 std::vector<Door> Room::getDoors(){
 	return myDoors;
 }
+
+std::vector<HolyWater> Room::getWaters(){
+	return myWaters;
+}
+
+DialogueBox Room::getDiaogueBox(){
+	return dialoguebox;
+}
+
+void Room::createDialogueBox(){
+	int sampleLimit = 2;
```

Comment:
the room shouldn't be creating the dialogue box. This should be done in GameView